cg/services/illumina_services/illumina_clean_sequencing_run_service.py

```diff
@@ -14,54 +14,63 @@
     HousekeeperFileMissingError,
 )
 from cg.models.run_devices.illumina_run_directory_data import IlluminaRunDirectoryData
-from cg.store.models import Flowcell, SampleLaneSequencingMetrics
+from cg.store.models import (
+    Flowcell,
+    SampleLaneSequencingMetrics,
+    IlluminaSequencingRun,
+    IlluminaSampleSequencingMetrics,
+)
 from cg.store.store import Store
 from cg.utils.files import remove_directory_and_contents
 from cg.utils.time import is_directory_older_than_days_old
 
 LOG = logging.getLogger(__name__)
 
 
-class CleanFlowCellAPI:
+class IlluminaCleanSequencingRunsService:
     """
-    Handles the cleaning of flow cells in the flow cells and demultiplexed runs directories.
+    Handles the cleaning of illumina sequencing runs in the sequencing-runs and demultiplexed-runs directories.
     Requirements for cleaning:
-            Flow cell is older than 21 days
-            Flow cell is backed up
-            Flow cell is in StatusDB
-            Flow cell has sequencing metrics in StatusDB
-            Flow cell has fastq files in Housekeeper or
-                Flow cell has SPRING files in Housekeeper
+            Sequencing run is older than 21 days
+            Sequencing run is backed up
+            Sequencing run is in StatusDB
+            Sequencing run has sequencing metrics in StatusDB
+            Sequencing run has fastq files in Housekeeper or
+                Sequencing run has SPRING files in Housekeeper
                 and
-                Flow cell has SPRING metadata in Housekeeper
-            Flow cell has a sample sheet in Housekeeper
+                Sequencing run has SPRING metadata in Housekeeper
+            Sequencing run has a sample sheet in Housekeeper
     """
 
     def __init__(
-        self, status_db: Store, housekeeper_api: HousekeeperAPI, flow_cell_path: Path, dry_run: bool
+        self,
+        status_db: Store,
+        housekeeper_api: HousekeeperAPI,
+        sequencing_run_path: Path,
+        dry_run: bool,
     ):
         self.status_db: Store = status_db
         self.hk_api: HousekeeperAPI = housekeeper_api
-        self.flow_cell = IlluminaRunDirectoryData(sequencing_run_path=flow_cell_path)
+        self.seq_run_dir_data = IlluminaRunDirectoryData(sequencing_run_path=sequencing_run_path)
         self.dry_run: bool = dry_run
-        LOG.info(f"Trying to delete {flow_cell_path}")
+        LOG.info(f"Trying to delete {sequencing_run_path}")
 
-    def delete_flow_cell_directory(self) -> None:
+    def delete_sequencing_run_directory(self) -> None:
```

Comment:
The current name gives the idea that it only deletes the sequencing dir
```suggestion
    def delete_run_directory(self) -> None:
```